pages/common/git-fork.md

```diff
@@ -0,0 +1,13 @@
+# git fork
+
+> Fork the given GitHub repo. Like `git clone` but forks first.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-fork>.
+
+- Fork and clone a GitHub repository by its URL:
+
+`git fork {{https://github.com/tldr-pages/tldr}}`
+
+- Fork and clone a specific GitHub repository by its 'slug':
```

Comment:
@mfrw, as you pointed out [here](https://github.com/tldr-pages/tldr/pull/6013#discussion_r637263949), `specific` is rather redundant. Also why put 	`slug` in `'..'`?

Reply:
@marchersimon Yes I had pointed that out, but when i thought about it;
- `slug` is the proper word for it.
- Looking at examples it is clear that it is the `username/reponame` == `slug`
-  We get to introduce a new term to the users in addition to provide what they came for :)
This way we are doing 2 things :)


in any case please let me know your thoughts. If we think its too specific, we can always change it.

Reply:
I think @marchersimon meant why `its 'slug'` as opposed to `its slug`.

Reply:
Ohhhh myyy :( 
silly me